dev/package.json

```diff
@@ -65,4 +65,4 @@
         "ts-loader": "^5.3.3",
         "webpack": "5.28.0"
     }
-}
+}
```

Comment:
Ok this is a super-nit, but if you don't fix it now then it'll keep happening. 

The ending newline was removed from the file, is VS Code doing this automatically? If so, then we'll want to try to turn that off.